src/api/env.production

```diff
@@ -12,11 +12,71 @@ COMPOSE_PROJECT_NAME=telescope_api
+# NOTE: if you change this, change all other occurrences below too.
 API_HOST=api.telescope.cdot.systems
 
 # The API Version, used as a prefix on all routes: /v1
 API_VERSION=v1
 
+
+################################################################################
+# Auth Service
+################################################################################
+
+# Image Service Port (default is 4444)
+AUTH_PORT=7777
+
+# Auth Service URL
+AUTH_URL=https://api.telescope.cdot.systems/v1/auth
+
+# The Single Sign On (SSO) login service URL
+SSO_LOGIN_URL=https://login.microsoftonline.com/eb34f74a-58e7-4a8b-9e59-433e4c412757/saml2
+
+# The callback URL endpoint to be used by the SSO login service (see the /auth route)
+SSO_LOGIN_CALLBACK_URL=https://login.microsoftonline.com/3ddc4284-8377-4878-94bf-a35840ca61fe/saml2
+
+# The Single Logout (SLO) service URL
+SLO_LOGOUT_URL=https://login.microsoftonline.com/3ddc4284-8377-4878-94bf-a35840ca61fe/saml2
+
+# The callback URL endpoint to be used by the SLO logout service (see the /auth route)
+SLO_LOGOUT_CALLBACK_URL=https://api.telescope.cdot.systems/v1/auth/logout/callback
+
+# The SSO Identity Provider's public key certificate. NOTE: this is the public
+# key cert of the test login IdP docker container.  Update for staging and prod.
+SSO_IDP_PUBLIC_KEY_CERT=MIIDXTCCAkWgAwIBAgIJALmVVuDWu4NYMA0GCSqGSIb3DQEBCwUAMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwHhcNMTYxMjMxMTQzNDQ3WhcNNDgwNjI1MTQzNDQ3WjBFMQswCQYDVQQGEwJBVTETMBEGA1UECAwKU29tZS1TdGF0ZTEhMB8GA1UECgwYSW50ZXJuZXQgV2lkZ2l0cyBQdHkgTHRkMIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAzUCFozgNb1h1M0jzNRSCjhOBnR+uVbVpaWfXYIR+AhWDdEe5ryY+CgavOg8bfLybyzFdehlYdDRgkedEB/GjG8aJw06l0qF4jDOAw0kEygWCu2mcH7XOxRt+YAH3TVHa/Hu1W3WjzkobqqqLQ8gkKWWM27fOgAZ6GieaJBN6VBSMMcPey3HWLBmc+TYJmv1dbaO2jHhKh8pfKw0W12VM8P1PIO8gv4Phu/uuJYieBWKixBEyy0lHjyixYFCR12xdh4CA47q958ZRGnnDUGFVE1QhgRacJCOZ9bd5t9mr8KLaVBYTCJo5ERE8jymab5dPqe5qKfJsCZiqWglbjUo9twIDAQABo1AwTjAdBgNVHQ4EFgQUxpuwcs/CYQOyui+r1G+3KxBNhxkwHwYDVR0jBBgwFoAUxpuwcs/CYQOyui+r1G+3KxBNhxkwDAYDVR0TBAUwAwEB/zANBgkqhkiG9w0BAQsFAAOCAQEAAiWUKs/2x/viNCKi3Y6blEuCtAGhzOOZ9EjrvJ8+COH3Rag3tVBWrcBZ3/uhhPq5gy9lqw4OkvEws99/5jFsX1FJ6MKBgqfuy7yh5s1YfM0ANHYczMmYpZeAcQf2CGAaVfwTTfSlzNLsF2lW/ly7yapFzlYSJLGoVE+OHEu8g5SlNACUEfkXw+5Eghh+KzlIN7R6Q7r2ixWNFBC/jWf7NKUfJyX8qIG5md1YUeT6GBW9Bm2/1/RiO24JTaYlfLdKK9TYb8sG5B+OLab2DImG99CJ25RkAcSobWNF5zD0O6lgOo3cEdB/ksCq3hmtlC/DlLZ/D8CJ+7VuZnS1rR2naQ==
+
+# Our apps's Entity ID, which is also the URL to our metadata.
+SAML_ENTITY_ID=https://api.telescope.cdot.systems/v1/auth/sp
+
+# SECRET = cookie session SECRET. If left empty, one will be set automatically
+SECRET=secret-sauce
+
+# ADMINISTRATORS is a list (space delimited) of users who have administrator
+# rights. Use the user's nameID (user2@example.com) or hashed version of
+# nameID (2b3b2b9ce8).  Either will work.
+ADMINISTRATORS=user1@example.com
```

Comment:
Shouldn't we set this with real user IDs?

Reply:
Yeah, when we get @chrispinkney's user stuff integrated. For now this is parity with the old stuff.